mono_repo/lib/src/commands/travis.dart

```diff
@@ -257,8 +258,7 @@ List<String> _calculatePkgEntries(
     if (config.beforeScript != null) {
       addEntry(pkg, [
         'echo',
-        'echo -e "${_wrap(prettyAnsi, styleBold, "PKG: $pkg")}"',
-        'echo -e "  Running `${config.beforeScript}`"',
+        _safeEcho(prettyAnsi, styleBold, '$pkg: before_script'),
```

Comment:
You removed the beforeScript value and are instead just printing 'before_script'. Why not print out the actual script?

Reply:
AFAIK `before_script` could be actual shell commands, right? Many?

Reply:
I *think* it can only be a single shell command.

It seemed inconsistent that we are *adding* the printing of the actual task command, but *removing* the printing of the actual beforeScript command.

Reply:
See latest. Now support List or String. Print out everything nicely...